### PR TITLE
 have mock using distribution-gpg-keys

### PR DESCRIPTION
**Pull request by sergiomb2**
- It is much more organized , move 112 files form root to etc/mock/ 
- Add  koji local repo mock cfg ...
- bug fixes 
- Using new include statement in mock configurations .

### CODE REVIEW

File: round.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-ARCHES="i386 x86_64 aarch64 armhfp ppc ppc64 ppc64le s390x"
```
Comment (kwizart) [inline round.sh]: should be a separate commit, unrelated to the other side.

**Comment by kwizart** [comment]
This PR is a total mess. I NAK it.

You are mixing important bugfix, with unrelated changes, This is totally insane.
Don't merge that.

File: el-round.sh [hunk 2]
```diff
@@ -14,6 +14,9 @@ for arch in $ARCHES ; do
   flavour=epel
   ffver=$fver
   if [ ! -f /etc/mock/epel-${fver}-${arch}.cfg ] ; then
+    echo "doesnt exit ${etc_mock}/epel-${fver}-${arch}.cfg"
+    # removing obsoleted .cfg
+    rm -f fedora-${fver}-${arch}-${repo}.cfg
```
Comment (sergiomb2) [inline el-round.sh]: hum I just check and in https://github.com/rpmfusion-infra/mock-rpmfusion we have all .cfg files and master file doesn't exist than we should remove it from our repo . should he move to old ?
also I got one bug here shouldn't be -f /etc/mock should be -f ${etc_mock}

**Comment by sergiomb2** [comment]
it is a lot of commits , do you want a lot of PR ?

File: el-round.sh (hunk 2 above)
Comment (sergiomb2) [inline el-round.sh]: why testing epel and removing fedora yeah bug , 2nd bug
